README.md

```diff
@@ -39,6 +39,34 @@ The MIT License (MIT)
 
 # Changelog
 
+## 1.6.4
+
+- As ID build using `WERCKER_GIT_COMMIT`
+- fix typo
+
+## 1.6.4
+
+- Better hash formating (added # char)
+
+## 1.6.3
+
+- Using `WERCKER_RUN_ID` instead `WERCKER_BUILD_ID` that is only in build workflow
```

Comment:
Using `WERCKER_RUN_ID` instead of `WERCKER_BUILD_ID`as that is only available in the build pipeline